elisp/intero.el

```diff
@@ -518,7 +534,7 @@ If the problem persists, please report this as a bug!")))
                           (forward-char 1)
                           (looking-back "$(" 1))
                         (+ 2 (current-column))
-                      (if (looking-at "$(")
+                      (if (looking-at-p "$(")
```

Comment:
This is certainly a reasonable change semantically, but I'd be surprised if it affects performance. After all, `looking-at-p` is defined as
```
(let ((inhibit-changing-match-data t))
    (looking-at regexp))
```

Reply:
Well, the C implementation of `looking-at` does pay attention to this variable and does more work if it's `nil`. For instance, please check https://github.com/emacs-mirror/emacs/blob/master/src/search.c#L334 (and possibly implementation of `re_match_2` as well).

Reply:
Thanks, great to know.